NotifyBackend/NotifyMe/views/views.py

```diff
@@ -216,14 +226,13 @@ def patch(self, request):
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
         except Exception as e:
             logger.error(f"Unexpected error occured while updating Subscription data. ERROR: {e}")
-            return Response(f"UNEXPECTED_ERROR_WHILE_UPDATING_SUBSCRIPTION_DATA. ERROR: {e}",
+            return Response(f"UNEXPECTED_ERROR_WHILE_UPDATING_SUBSCRIPTION_DATA.",
             status=status.HTTP_500_INTERNAL_SERVER_ERROR)
              
 
     def delete(self, request):
```

Comment:
you are deleting directly in the view dont do this create a method in service and call that method to delete